python/revng/scripting/cli_project.py

```diff
@@ -0,0 +1,105 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import subprocess
+from pathlib import Path
+from typing import Any
+from revng.model import Binary, dump_diff, load_model_from_string, make_diff
+
+from .project import Project
+
+
+class CLIProject(Project):
+    def __init__(self, tmp_diff_dir: str = "/tmp"):
+        self.tmp_diff_path = Path(f"{tmp_diff_dir}/tmp-diff.yml")
+
+    def import_binary(self, working_dir: str, input_binary: str):
+        self.working_dir = working_dir
+        self.input_binary = input_binary
+
+        self.analyze("import-binary")
```

Comment:
Here there should be an invocation to the analyses lists `revng{,c}-initial-auto-analysis`